app/src/main/java/app/revanced/integrations/returnyoutubedislike/ReturnYouTubeDislike.java

```diff
@@ -219,9 +219,9 @@ private static void updateDislike(AtomicReference<Object> textRef, boolean isSeg
                 // discussion about this: https://github.com/Anarios/return-youtube-dislike/discussions/530
 
                 //
-                // Change the "Likes" string to show that likes are hidden
+                // Change the "Likes" string to show that likes and dislikes are hidden
                 //
-                newDislikeString = "Likes Hidden by Creator"; // for now, this is not localized
+                newDislikeString = "Likes Hidden"; // for now, this is not localized
```

Comment:
Not only the hides would be hidden, also the dislikes, which is why this should say "Hidden" only.

Reply:
changed to "Hidden"